appveyor.yml

```diff
@@ -40,7 +39,7 @@ build_script:
   - tar -zcvf windows.tar.gz spago.exe
 
 test_script:
-  - set PATH=C:\projects\spago\bin;%PATH%
+  - set PATH=C:\projects\spago\bin;C:\projects\spago\purescript;purescript;%PATH%
```

Comment:
extra `purescript;`, is this a typo?

Reply:
No, I didn't know which one of the two would work so I put both 😄

Reply:
Fixed 🙂